test-harness/helpers.ts

```diff
@@ -1,41 +1,19 @@
-import * as  fs from 'fs';
-import { join } from 'path';
-import fetch from 'node-fetch';
+export function parseSpecFile(spec: string): any {
+  const regex = /==(server|test|spec|command|expect)==\n(([^=])*)/gi
```

Comment:
Better to do four `====` because == could appear in the content of these things, and [^=] is even more likely to appear in the contents.

Did you try splitting instead of matching?

Reply:
Changed with 4; trying with splitting…